signature/src/signatures/sigstore.rs

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) 2021 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::image;
+use anyhow::{anyhow, Result};
+use oci_distribution::Reference;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::ffi::OsStr;
+use std::fs;
+
+// FIXME: Now, for Demo, the dir which the sigstore config files locate
+// is hardcoded to "/etc/containers/registries.d".
```

Comment:
I don't think the comment needs to repeat the code here:

```suggestion
// is hardcoded.
```